libraries/AP_Scripting/applets/Aerobatics/FixedWing/TrajectoryBased/Schedules/F4CScaleExample.txt

```diff
@@ -0,0 +1,178 @@
+#   trajectory tracking aerobatic control
+#   See README.md for usage
+#   Written by Matthew Hampsey, Andy Palmer and Andrew Tridgell, with controller
+#   assistance from Paul Riseborough, testing by Henry Wurzburg
+
+# To use this schedule put the file on your microSD in the root directory 
+# (not the APM directory) with name trick91.txt The schedule can then be 
```

Comment:
This seems to contradict the readme which says to put it in the scripts directory.

Reply:
historical from previous commits...not mine...but will take the opportunity to fix